Cargo.toml

```diff
@@ -11,12 +11,15 @@ members = [
     "zingolib",
     "zingoconfig",
     "zingo-testvectors",
+    "zingo-netutils",
     "zingo-memo",
 ]
 resolver = "2"
 
 [workspace.dependencies]
 clap = "4.4"
+bip0039 = "0.10.1"
```

Comment:
Is there not a re-export of this functionality in librustzcash..  perhaps under zip0039